src/fights.ts

```diff
@@ -846,12 +846,43 @@ const freeFightSources = [
   ),
 
   new FreeFight(
-    () => have($item`[glitch season reward name]`) && !get("_glitchMonsterFights"),
-    () => {
-      restoreHp(myMaxhp());
-      retrieveItem($item`[glitch season reward name]`);
-      visitUrl("inv_eat.php?pwd&whichitem=10207");
-      runCombat();
+    () =>
+      have($item`[glitch season reward name]`) &&
+      !get("_glitchMonsterFights") &&
+      get("garbo_fightGlitch", false),
+    () =>
+      withMacro(
+        Macro.skill($skill`Curse of Marinara`)
+          .trySkill($skill`Conspiratorial Whispers`)
+          .trySkill($skill`Shadow Noodles`)
+          .externalIf(
+            get("glitchItemImplementationCount") >= 1000,
+            Macro.item([$item`gas can`, $item`gas can`])
+          )
+          .externalIf(
+            get("lovebugsUnlocked"),
+            Macro.trySkill($skill`Summon Love Gnats`).trySkill($skill`Summon Love Mosquito`)
+          )
+          .trySkill($skill`Micrometeorite`)
+          .tryItem($item`Time-Spinner`)
+          .tryItem($item`little red book`)
+          .tryItem($item`Rain-Doh blue balls`)
+          .tryItem($item`Rain-Doh indigo cup`)
+          .trySkill($skill`Entangling Noodles`)
```

Comment:
Frost Bite is a reusable stun too, we might as well add it.
Also, shouldn't we still be using Sing Along, at least at lower implementation levels?

Reply:
I won't be adding sing along or any stasis to this personally, but Frost Bite is a good idea.